src/aurora/xmlfixer.cpp

```diff
@@ -110,7 +110,7 @@ Common::UString XMLFixer::fixXMLElement(const Common::UString element) {
 		Common::UString::iterator it2 = it1->findLast(' ');
 		name = it1->substr(it2, it1->end());
 		value = it1->substr(it1->begin(), it2);
-			
+		

```

Comment:
There's trailing whitespace here. We want empty lines to be completely empty, no even containing spaces or tabs.

And we want no spaces and tabs at the end of a line at all.

There are other places with those in the code, like in xmlfixer.h.